.github/workflows/docker-build.yaml

```diff
@@ -508,27 +589,31 @@ jobs:
                     mv docker-compose.cypress.yml docker-compose.yml
                     docker compose up -d --wait
                     docker compose exec php-fpm php phing -D production.confirm.action=y db-create frontend-api-generate-new-keys build-demo-dev-quick error-pages-generate elasticsearch-index-recreate elasticsearch-export
-            -   name: Run Cypress tests
+            -   name: Run regenerate Cypress tests for ${{ matrix.group }}
```

Comment:
when some test fails, the others are skipped (see https://github.com/shopsys/shopsys/actions/runs/11591269071/job/32270835776?pr=3542) 
How will this behave when regenerating screenshots?